content/en/docs/tasks/observability/telemetry/index.md

```diff
@@ -73,6 +73,10 @@ will be used when the `Telemetry` resources do not specify a provider.
 If you're using [Sidecar](/docs/reference/config/networking/sidecar/) configuration, do not forget to add provider's service.
 {{< /tip >}}
 
+{{< tip >}}
+Provider does not support `$(HOST_IP)`. If you're running collector in agent mode, you can [use service internal traffic policy](https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy/#using-service-internal-traffic-policy), set `InternalTrafficPolicy` to `Local` for better performance.
```

Comment:
which provider?

Reply:
> The Telemetry API uses the concept of providers to indicate the protocol or type of integration to use. Providers can be configured in [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider).

Reply:
i saw different providers above, so was wondering if the tip is for any specific provider or all of them in general.